recipe/meta.yaml

```diff
@@ -19,6 +19,7 @@ requirements:
     - python >=3.9
     - pip
     - setuptools
+    - setuptools_scm
```

Comment:
Does it matter if it's an underscore as opposed to a dash? (`setuptools-scm`)

Reply:
It looks like both work (geocat-comp has one and xarray the other option), but I was wondering the same thing.  

It seems others are confused by this as well: https://conda.discourse.group/t/dual-packages-underscore-vs-dash/328